site/data/smc.yaml

```diff
@@ -1,3 +1,10 @@
-# This file is automatically generated in .github/workflows/site.yaml
-- Name:
-  Company:
+- Name: Phillip Cloud
+  Association: Voltron Data
+- Name: Weston Pace
+  Association: LanceDB
+- Name: Jacques Nadeau
+  Association: Sundeck
+- Name: Victor Barua
+  Association: DataDog
```

Comment:
Thanks for making this change!

Minor request, could you use `Datadog` instead of `DataDog`. The official branding is the former (and yes it's not consistent with company github handle)

Reply:
Absolutely! Good catch